pkg/auth/rbac.go

```diff
@@ -33,41 +33,33 @@ import (
 type RBACConfig struct {
 	// Indicates if Dex is enabled.
 	DexEnabled bool
-	// The RBAC policy. Key is for example "p,Developer,EnvironmentLock,Create,production,allow"
-	Policy map[string]*Permission
+	Policy     map[string]*Permission
```

Comment:
why delete the comment? Either keep the comment or explain in other words what the key of this map is.